Documentation/Signoffs/Raymond Mule-Signoff-Wireless Communication Subsystem(Updated).md

```diff
@@ -0,0 +1,73 @@
+| 3. The entire subsystem shall weigh less than 1.2 pounds| From conceptual design |
+| 4. Unlicensed devices that operate in the 915 MHz band allow for a maximum power output of up to 30 dBm or 1 Watt | From FCC Part 15 Subpart C |
+| 5. The bandwidth of the device must be 125 kHz or 500 kHz | From further analysis of the subsystem |
+| 6. The system must be able to transmit a data packet at the size of at least 30 bits of data | From further analysis of the subsystem |
+| 7. The system must be compatible with Linux based computing systems | From further analysis of the subsystem |
+
+Since DARPA does not specifiy a minimum distance that the drone can be away from the command center, a value had to be calculated to determine this distance. This calculation was found by analyzing the drone chosen to hold the entire sensing system from the conceptual design. Assuming the drone holds the max payload of six pounds, the drone has a battery life of approximately thirty minutes. The max speed of the drone while carryign the max payload of six pounds as well as some consideration for wind resistance is approximately ten meters-per-second. In this calculation, it will be assumed that the drone will cover a circular area with the command center of the drone being at the center of the circle. The maximum distance the drone can travel with a maximum payload and a speed of ten meters-per-second while also being able to return to the command center with one charge of the battery is nine kilometers.
+
+The RAK811 transceiver has a very high sensitivity of -148 dBm, enabling it to transmit data over extremely long distances. This high sensitivity allows the transceiver to receive signals that have become very weak after traveling a great distance.
+
+## Schematic- 
+
+![Newest Schematic](https://github.com/Michaelwwest98/DARPA-Drone-Triage-Sensing-System/assets/123600489/cc0cd754-5744-4b14-8777-ad1688809484)
+
+| RAK811 LoRa Breakout Board PINOUT |---|
+|-----------------------------------|---|
+| Pin Number | Description |
+| 1 | 3.3 V |
+| 2 | UART1_TXD |
+| 3 | UART1_RXD |
+| 4 | PB12 |
+| 5 | RST |
+| 6 | PA3 |
+| 7 | PB5 |
+| 8 | PA15 |
+| 9 | BOOT0 |
+| 10 | GND |
+| 11 | PA0 |
+| 12 | PA1 |
+| 13 | SWCLK |
+| 14 | SWDIO |
+| 15 | PB11 |
+| 16 | PB15 |
+| 17 | PA2 |
+| 18 | PA13 |
+| 19 | PA12 |
+| 20 | PB14 |
+
+## Analysis-
+1.  The RAK811 transceiver has a range of 15 kilometers, which fulfills the constraint of a 9 kilometer distance.
+
+2. The data that will be sent from the computing system to the user interface will be approximately 30 bits in size according to the designer of the computing subsystem. The RAK811 has an average data rate of 300 kbps meaning that the data from the computing subsystem will be able to reach the user interface subsystem within the two second minimum time frame specified in the constraints.
```

Comment:
How do we know? How long will it take at maximum between receiving two packets of data?